test/e2e/policySet_e2e_test.go

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/controllers/common"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	testcommon "github.com/stolostron/governance-policy-framework/test/common"
+)
+
+const (
+	testPolicyName          string = "test-policy"
+	testPolicySetName       string = "test-policyset"
+	testPolicyYaml          string = "../resources/policy_set/test-policy.yaml"
+	testPolicySetPatchYaml  string = "../resources/policy_set/patch-policy-set.yaml"
+	testedDisablePolicyYaml string = "../resources/policy_set/disable-policy.yaml"
+)
+
+var _ = Describe("Test policy set", func() {
+	Describe("Create policy, policyset, and placement in ns:"+userNamespace, func() {
+		It("Should create and process policy and policyset", func() {
+			By("Creating " + testPolicyYaml)
+			output, err := utils.KubectlWithOutput("apply",
+				"-f", testPolicyYaml,
+				"-n", userNamespace,
+				"--kubeconfig=../../kubeconfig_hub")
+			By("Creating " + testPolicyYaml + " result is " + output)
+			Expect(err).To(BeNil())
+
+			plc := utils.GetWithTimeout(
+				clientHubDynamic, testcommon.GvrPolicy, testPolicyName, userNamespace, true, defaultTimeoutSeconds,
+			)
+			Expect(plc).NotTo(BeNil())
+
+			By("Patching " + testPolicyName + "-plr with decision of cluster managed")
+			plr := utils.GetWithTimeout(
+				clientHubDynamic, testcommon.GvrPlacementRule, testPolicySetName+"-plr", userNamespace,
+				true, defaultTimeoutSeconds,
+			)
+			plr.Object["status"] = utils.GeneratePlrStatus(clusterNamespace)
+			_, err = clientHubDynamic.Resource(testcommon.GvrPlacementRule).Namespace(userNamespace).UpdateStatus(
+				context.TODO(), plr, metav1.UpdateOptions{},
+			)
+			Expect(err).To(BeNil())
+
+			By("Checking " + testPolicyName + " on managed cluster in ns " + clusterNamespace)
+			managedplc := utils.GetWithTimeout(
+				clientHubDynamic, testcommon.GvrPolicy, userNamespace+"."+testPolicyName, clusterNamespace, true,
+				defaultTimeoutSeconds,
+			)
+			Expect(managedplc).NotTo(BeNil())
+
+			opt := metav1.ListOptions{
+				LabelSelector: common.RootPolicyLabel + "=" + userNamespace + "." + testPolicyName,
+			}
+
+			By("Patching replicated policy statuses - NonCompliant")
```

Comment:
Patching the status of replicated policy is not needed. This is running against kind clusters with all grc controller installed so the status will be generated by controller.

Reply:
Should just check the status of replicated policy